url.bs

```diff
@@ -246,9 +305,28 @@ a <var>percentEncodeSet</var>, run these steps:
    <td>"<code>‽%25%2E</code>"
    <td>0xE2 0x80 0xBD 0x25 0x2E
   <tr>
-   <td><a for="code point">UTF-8 percent-encode</a> <var>input</var> using the
+   <td rowspan=3><a for="code point">Percent-encode after encoding</a> with <a>Shift_JIS</a>,
```

Comment:
I'd like to see @rmisev 's example added here, since it's a key case (i.e., all of these examples present would work in your old algorithm and your new one; your fixed algorithm differs only by that special case where some of the bytes that encode the code point are < 128, which seems to happen in ISO-2022-JP but not UTF-8 or Shift-JIS).

So, the example is:
- Operation: Percent-encode after encoding with ISO-2022-JP, _input_, and the userinfo percent-encode set
- Input: U+00A5 (¥)
- Output: "`%1B(J\%1B(B`"